examples/sdk/node/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import { BacktraceClient } from '@backtrace/node';
+import fs from 'fs';
+import path from 'path';
+import { exit } from 'process';
+import readline from 'readline';
+import { SUBMISSION_URL } from './consts';
+const reader = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+const client = BacktraceClient.builder({
+    url: SUBMISSION_URL,
+    attachments: [path.join(path.dirname(process.cwd()), 'samplefile.txt')],
+    rateLimit: 5,
+    userAttributes: {
+        'custom-attribute': 'test',
+        'custom-annotation': {
+            prop1: true,
+            prop2: 123,
+        },
+    },
+}).build();
+
+console.log('Welcome to the @Backtrace/node demo');
+const menu = `Please pick one of available options:
+1. Send an exception
+2. Send a message
+0. Exit
+    
+Type the option number:`;
```

Comment:
Can we move this variable to `showMenu` function?

Reply:
done.